angular/tasks/app.coffee

```diff
@@ -19,7 +19,11 @@ module.exports = ->
       htmlmin(),                                  # minify html
       template(                                   # store html templates in angular cache
         module: 'loomioApp',
-        transformUrl: (url) -> "generated/components/#{url}"
+        transformUrl: (url) ->
```

Comment:
Yep, sure. It's a hacky workaround through the angular router, which I'm hoping to find a better solution for.

Basically the templates which come through the core app are getting the format `generated/components/my_component/my_component.html`, while the plugin ones are getting `generated/components/my_component.html`

That's all fine until you go to add a new component to the router, which insists that this:
```
{path: '/tags', components: 'tagsPage'}
```
map to 
```
generated/components/tags_page/tags_page.html
```

which I haven't found an override for yet. Thus this hacks the plugin templates to use the intended component template url.